test/testinput/bufr_ncep_adpsfc_nc000000.yaml

```diff
@@ -0,0 +1,188 @@
+# (C) Copyright 2020 NOAA/NWS/NCEP/EMC
+# #
+# # This software is licensed under the terms of the Apache Licence Version 2.0
+# # which can be obtained at http://www.apache.org/licenses/LICENSE-2.0.
+#
+observations:
+  - obs space:
+      name: bufr
+
+      obsdatain: "./testinput/gdas.t12z.adpsfc.nc000000_thinned.tm00.bufr_d"
+
+      exports:
+        variables:
+          timestamp:
+            datetime:
+              year: "*/YEAR"
+              month: "*/MNTH"
+              day: "*/DAYS"
+              hour: "*/HOUR"
+              minute: "*/MINU"
```

Comment:
If you only want NC000000 data here then you should specify NC000000 as the subset here as well. This currently works because you pre-split your file... So these queries are inconsistent with the other queries in the file.

Reply:
Fair enough. Fixing now.